base.cfg

```diff
@@ -0,0 +1,107 @@
+[buildout]
+show-picked-versions = true
+extensions =
+    mr.developer
+
+index = https://pypi.python.org/simple/
```

Comment:
This is outdated. Just remove the index line. Buildout already knows where pypi is.

Reply:
Yes. I just used the `bobtemplates.plone` configuration:

https://github.com/plone/bobtemplates.plone/blob/0b86c3771dd9d7aa9d253fa9ec1fa07f5254d193/bobtemplates/plone/addon/base.cfg.bob#L6

I think they use it there, because the Plone 4.3 buildout uses an old buidout:

https://github.com/plone/bobtemplates.plone/blob/8a41b48530b2c7c653923a541c249e00f13f8b5d/bobtemplates/plone/addon/test_plone43.cfg#L11

I think the Plone 4.3 buildout version should be updated there. As here I updated the buildout version, I will remove this line.

Reply:
Oh right, maybe that buildout version was still pointing to `index = http://pypi.python.org/simple/` by default.
Anyway https://pypi.org/simple/ would be better IMO... I opened https://github.com/plone/bobtemplates.plone/issues/436